Source/Lib/Fluxor.Blazor.Web.Analyzers/Fluxor.Blazor.Web.Analyzers.csproj

```diff
@@ -13,7 +13,7 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.9.2" PrivateAssets="all" />
+		<PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" PrivateAssets="all" />
```

Comment:
This is necessary because `Workspaces` aren't allowed in analyzers. It's very easy to accidentally depend on an API that will blow up in `dotnet build` but not in the IDE.